justfile

```
@@ -234,7 +234,11 @@ addnoun noun:
     exit 0
   fi
 
-  echo "{{noun}}/SM" >> $DICT_FILE
+  if [[ "{{noun}}" =~ ^[A-Z] ]]; then
+    echo "{{noun}}/M" >> $DICT_FILE
+  else
+    echo "{{noun}}/SM" >> $DICT_FILE
+  fi
```

You should comment the code in the very same way you explained in the PR and commit

Or at least mention the PR I'd

Please note, the notation /SM or /M obscure for people

Also, I would recommend something like this


```suggestion
  modifier="SM" # a comment explaining what SM is about
  if [[ "{{noun}}" =~ ^[A-Z] ]]; then
    modifier="M" # a comment explaining how it differs
  fi
  echo "{{noun}}/$modifier" >> $DICT_FILE
```

This is pseudo code, please adapt